docs/concepts.rst

```diff
@@ -1,119 +1,108 @@
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Project Directory
+^^^^^^^^^^^^^^^^^
 
-``cumulusci.yml`` is the definition of the project's automation. It includes information about the project itself as well as customizations that define how scratch orgs are built and provide other important functionality to be used as part of the project's lifecycle.
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, you can always be confident of the scope of the actions you're taking. And thanks to the keychain, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-When you work with CumulusCI in a project, the contents of your ``cumulusci.yml`` are merged with the ``cumulusci.yml`` that comes with CumulusCI and defines out-of-the-box functionality. From time to time we'll refer to that as the standard library. Learn more about customizing CumulusCI automation in TODO: link Customization section.
+[NOTE: Save yourself a headache by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.)]
 
-Project Source Code
-^^^^^^^^^^^^^^^^^^^
+In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the [TODO link Get Started section].
 
-The main body of the project's code and metadata lives in the ``force-app`` directory for Salesforce DX-format projects and ``src`` for Metadata API-format projects. For managed package projects, only this metadata is part of the package. Most projects have the vast majority of their components stored here.
+cumulusci.yml
+^^^^^^^^^^^^^
 
-Orgs
-^^^^
+The ``cumulusci.yml`` file defines a project's automation in CumulusCI. It contains all the customizations and configurations that pertain to your project's lifecycle. It can encompass everything from customizing the shapes of scratch orgs to configuring tasks and flows.
 
-The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in TODO: link Scratch Org Management. 
+Learn more about customizing CumulusCI automation in the [TODO: link Customization section].
 
-Datasets
+force-app (or src)
+^^^^^^^^^^^^^^^^^^
+
+The main body of the project's code and metadata lives in the ``force-app`` directory for Salesforce DX-format projects and ``src`` for Metadata API-format projects. For managed package projects, only this metadata is part of the package. Projects have the vast majority of their components stored here.
```

Comment:
"For managed package projects, only this metadata is part of the package."

A bit vague. Should we clarify further?

Reply:
What do you think needs clarification? I think the point of the sentence is to contrast it with other sets of metadata in the unpackaged folder.

Reply:
@davisagli I think I need a better understanding then of where metadata is stored. To start: A managed package also has an unpackaged folder?

Reply:
@justinvholt It can. There's the primary package of metadata in force-app which is going to be released as a managed package, and then there can be other sets of metadata in various folders under `/unpackaged`. These can serve a variety of different purposes such as:
- occasionally it's necessary to deploy some metadata in order to enable a feature that is a prerequisite for installing a managed package
- there are some types of metadata that aren't supported in managed packages, but can still be deployed as unmanaged metadata
- there can be metadata which only needs to be deployed for particular configurations, such as some additional fields that are used for QA purposes but not wanted in other types of orgs

Reply:
So is it that the unmanaged metadata that's part of a package is only used as a means to deploy/install the managed package but not "officially" part of (supported by?) the managed package once it's installed? (Does this question make sense?)

Is there a better way to convey how integral force-app folder/this metadata is to a managed package?

Reply:
Unmanaged metadata is, by definition, not part of the managed package, but it can still be an important part of delivering a full product to customers. The end goal is not to install a package; the end goal is to deliver a fully configured product. Sometimes doing that requires deploying unmanaged metadata before or after the managed package is installed.

> Is there a better way to convey how integral force-app folder/this metadata is to a managed package?

The default package directory (usually called force-app) defines what will be included when you release a managed package from your cci project (or when you release an unlocked package, if you're doing that, or if you are not releasing the package at all but running the `deploy` task to get the metadata into an org in unmanaged form). It's the right place to put metadata except for when there's a reason to put it somewhere else (like one of the reasons I listed above).